srt.go

```diff
@@ -34,17 +34,30 @@ func ReadFromSRT(i io.Reader) (o *Subtitles, err error) {
 	// Scan
 	var line string
 	var lineNum int
+
+	// For the Index Counter
+	saveIndex := true
+	var index int
+	var convErr error
+
 	var s = &Item{}
 	for scanner.Scan() {
 		// Fetch line
 		line = strings.TrimSpace(scanner.Text())
+
+		// Fetch Index
+		if saveIndex {
+			index, convErr = strconv.Atoi(strings.Replace(line, "\ufeff", "", -1))
```

Comment:
Please remove the BOM header this way:

```go
// Remove BOM header
if lineNum == 1 {
    line = strings.TrimPrefix(line, string(BytesBOM))
}
```

Reply:
@asticode True that. better to remove the BOM header using lineNum variable.